analyzer/codechecker_analyzer/analyzers/clangsa/config_handler.py

```diff
@@ -28,7 +28,6 @@ def __init__(self, environ):
         super(ClangSAConfigHandler, self).__init__()
         self.ctu_dir = ''
         self.ctu_on_demand = False
-        self.log_file = ''
```

Comment:
This member was stored in the `construct_config_handler()` function, but it is read nowhere. Since the store operation has been removed, this variable became completely unused.